C7/MapView.cs

```diff
@@ -983,6 +983,79 @@ public override void drawObject(LooseView looseView, GameData gameData, Tile til
 	}
 }
 
+public class RoadLayer : LooseLayer {
```

Comment:
Optional: I've been trying to encourage keeping our file sizes moderate, and in that interest the ResourceLayer.cs file (C7/Map/ResourceLayer.cs) is the first map layer to be in the new Map folder.  Refactoring hasn't been a high priority since the map has been stable for a while, but long-term it would be nice to have the classes for drawing layers live in separate files in that folder, and the MapView live there too.  Wouldn't suggest doing all that here, but maybe the new class living in that folder.

Reply:
Rider has refactoring extracting classes to separate files in directory. I extracted RoadLayer only, worked nice.

Reply:
A fellow Rider user!  Good to know I'm not the only one anymore!  I don't think I knew about that refactoring option though, will have to keep it in mind for next time.